client/my-sites/invites/invite-accept-logged-out/index.jsx

```diff
@@ -50,7 +50,7 @@ class InviteAcceptLoggedOut extends Component {
 		window.location = signInLink;
 	};
 
-	submitForm = ( form, userData, _, afterSubmitCallback = noop ) => {
```

Comment:
Form was not being used at all, removing

Reply:
Since we can't change the `submitForm` like this, you can simply use `_` instead of form in the parameters

Reply:
Added back.